pkg/controllers/federatedcluster/util_test.go

```diff
@@ -260,7 +260,8 @@ func Test_aggregateResources(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			allocatable, available := aggregateResources(tc.nodes, tc.pods)
```

Comment:
let's add a test case for filtering nodes by label

Reply:
ok